ntp/syncTime.go

```diff
@@ -195,17 +197,19 @@ func (s *syncTime) sync() {
 
 	clockOffsetsWithoutEdges := s.getClockOffsetsWithoutEdges(clockOffsets)
 	clockOffsetHarmonicMean := s.getHarmonicMean(clockOffsetsWithoutEdges)
+	if clockOffsetHarmonicMean < -outOfBoundsDuration || clockOffsetHarmonicMean > outOfBoundsDuration {
```

Comment:
why don't you use abs ?

Reply:
done, however, I have created a function for Abs because math.Abs operates on floats.